source/exploring_catalog_and_datasets/04_getting_involved/using_api.rst

```diff
@@ -3,14 +3,9 @@ Using the API
 
 An API (Application Programming Interface) is an interface for programs to communicate and exchange data.
 
-.. admonition:: Note
-   :class: note
-
-   To know more about what APIs are and why and how they are used, you can read the article "`You don\'t need analogies to understand APIs <https://kitchen.opendatasoft.com/you-dont-need-analogies-to-understand-apis-14da4513f970>`_" written by one of our Opendatasoft developers.
```

Comment:
Why removing the link to Julien's article? I find it very clear as a first jump into _what APIs are_.